app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -63,3 +63,42 @@ input.search-bar {
   background: white url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' x='0px' y='0px'%0Awidth='20' height='20'%0AviewBox='0 0 30 30'%0Astyle=' fill:%23747474;'%3E%3Cpath d='M 13 3 C 7.4889971 3 3 7.4889971 3 13 C 3 18.511003 7.4889971 23 13 23 C 15.396508 23 17.597385 22.148986 19.322266 20.736328 L 25.292969 26.707031 A 1.0001 1.0001 0 1 0 26.707031 25.292969 L 20.736328 19.322266 C 22.148986 17.597385 23 15.396508 23 13 C 23 7.4889971 18.511003 3 13 3 z M 13 5 C 17.430123 5 21 8.5698774 21 13 C 21 17.430123 17.430123 21 13 21 C 8.5698774 21 5 17.430123 5 13 C 5 8.5698774 8.5698774 5 13 5 z'%3E%3C/path%3E%3C/svg%3E") no-repeat 13px center;
 }
 
+.icon-circle {
+  width: 100px;
+  height: 100px;
+  background-color: $primary-light;
+}
+
+//Heroicons
+.w-18 {
+  width: 18px;
+}
+
+//default Heroicon width
+.w-24 {
+  width: 24px;
+}
+
+.w-30 {
+  width: 30px;
+}
+
+.w-36 {
+  width: 36px;
+}
+
+.w-48 {
+  width: 48px;
+}
+
+.w-60 {
+  width: 60px;
+}
+
+.w-66 {
+  width: 66px;
+}
+
+.w-72 {
+  width: 72px;
+}
```

Comment:
This seems weird - I'd rather we define 3 or 4 sizes (S, M, L, XL) with preset widths to help with consistency

Reply:
It works really nice tho.
Heroicons are made to work with Tailwind CSS, which have all those classes.
I tried with 3-4 size but definitely doesn't work as good.

Reply:
Then I would predefine (hi-xs, hi-s, hi-m, hi-l, hi-xl) rather than defining `w-24` and what not